R/generate_qc_report.R

```diff
@@ -23,6 +27,8 @@ generate_qc_report <- function(library_id,
                                unfiltered_sce,
                                filtered_sce = NULL,
                                processed_sce = NULL,
+                               rmd_file = NULL,
```

Comment:
I'd suggest renaming with `template` in the name for clarity. Maybe not exactly this name, but something to say it is a template file.
```suggestion
                               report_template = NULL,
```